src/components/GameList/GameListFilter.svelte

```diff
@@ -1,9 +1,113 @@
 <script>
+	import FilterPopUp from '$lib/GameList/GameListPopup.svelte';
```

Comment:
Dit moet wel even anders vind ik. Is niet heel consistent

`GameListPopup` wordt `FilterPopUp`. Ben altijd wel voorstander om hetgeen wat je importeert hetzelfde te noemen als hoe je het in het component gebruikt. Nu is `GameListPopup` misschien niet zo'n goede naam, maar weet niet. Misschien dat je het op een andere plek kan zetten, want dit vind ik nogal verwarren.d